geom/type_geometry_collection.go

```diff
@@ -234,3 +240,134 @@ func (c GeometryCollection) Area() float64 {
 	}
 	return sum
 }
+
+func highestDimensionIgnoreEmpties(geom Geometry) int {
```

Comment:
Nit: using `geom` as the variable name is the same as the package name, which may be a bit confusing. Thoughts on just using `g` instead? Most other places in the code use `g` as the param for a `Geometry`.

Reply:
Changed all instances of `geom` as a variable name in this file into `g` instead, including in other funcs.